hunter: initial rerender

ke-zhang-rd opened the pull request:
https://github.com/ionelmc/python-hunter
https://pypi.org/project/hunter/

Commit:
MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.01

mrakitin committed:
Add setuptools into the host reqs

mrakitin commented:
I figured out it cannot be a noarch package as it's heavily using cython and C-extensions.

Commit:
MNT: it cannot be a noarch package as it's heavily using cython and C-extensions.

mrakitin committed:
Add compilers & rerender

mrakitin committed:
MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.08

mrakitin committed:
Add manhole run dependency

mrakitin committed:
Exclude manhole for win

mrakitin committed:
Use nsls-ii-forge's conda-forge.yml

mrakitin committed:
MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.08

mrakitin committed:
Use c/cxx compilers for OSX of version 4 & rerender

MNT: Re-rendered with conda-build 3.18.9, conda-smithy 3.5.0, and conda-forge-pinning 2019.10.08

mrakitin commented:
A temporary workaround is implemented in https://github.com/nsls-ii-forge/hunter-feedstock/pull/1/commits/9cfa850896475968facc63399bc2c610fa541907 before we have the compilers of v9 for OSX.

All builds are successful, thanks for the contribution, @ke-zhang-rd! I think we can promote the packages to conda-forge.
